### DISCUSSION TITLE
how to deny file types is not documented, how can I do that

### DISCUSSION THREAD

**Discussion by PackElend** [drakkan/sftpgo#1315] [Q&A]
Hi there,
I'm pretty new to sftpgo. I'm currently playing around with it, desperately trying to figure out how to deny a user's upload of a ".exe" file.

I have been experimenting with the ACL of the user but nothing works, can someone help out?

I'm running 
SFTPGo 2.5.1-5b4a1bda-2023-05-20T17:12:44Z +metrics +azblob +gcs +s3 +bolt +mysql +pgsql +sqlite -unixcrypt +portable
on Windows 10 

by the way 
"C:\ProgramData\SFTPGo\sftpgo.json"
does not contain any user configuration information 

thank you

**Comment by drakkan** [comment]
Hello,

you have to configure a per-directory pattern restriction. I'm sorry but I no longer provide step-by-step support. Please check the [support policy](https://github.com/drakkan/sftpgo#support-policy). Thank you

**Comment by PackElend** [comment]
I don't know what I have done wrong before but it works :)

blocking .exe globally for the user
![image](https://github.com/drakkan/sftpgo/assets/3542031/9f70ad45-bab8-4a60-9d26-07fa8a828ade)

that would allow it again 
![image](https://github.com/drakkan/sftpgo/assets/3542031/9d13f16d-a8eb-4c50-9236-33e365f344a5)
